index.html

```diff
@@ -48,7 +57,180 @@ <h1>I’m Nurbol<br>Glad to see<br> you!</h1>
+                <li>css</li>
+                <li>javaScript</li></ul>
+                <button class="action">See Project</button>
+            </div>
+        </div>
+    </div>            
+</div>    
+    </section>
+
+<!-- ─── About Me Section ─── -->    
+
+    <section class="aboutme">
+        <div class="text">
+            <h1>About<br>Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class=links>
+            <button type="button" class="button">LET’S CONNECT</button>
+            <ul class="media-links">
+                <li class="media"><a href="https://twitter.com/everinurmind/"><img src="images\Twitter@2x.png" alt="twitter logo"></a></li>
+                
+                <li class="media"><a href="https://www.linkedin.com/in/everinurmind/"><img src="images\Linkedin@2x.png" alt="linked-in logo"></a></li>
+                
+                <li class="media"><a href="https://medium.com/"><img src="images\Medium@2x.png" alt="medium logo"></a></li>
+                
+                <li class="media" ><a href="https://github.com/everinurmind/"><img src="images\GitHub@2x.png" alt="github logo"></a></li>
+                
+                <li><a class="media" href="https://www.angellist.com/"><img src="images\AngelList@2x.png" alt="angellist logo"></a>
+                </li>
+            </ul>
+            <button class="action2">Get My Resume</button>
+        </div>
+        <div class="abtme">
+        <div class="languages">
+                <div id="intro">
+                    <h3>Languages</h3>
+                    <button class="arrow"><img src="images\Union@2x.png"></button>
+                </div>    
+            <ul class="slide">
+                <li>
+                    <img src="worksimg\Ellipse 1@2x.png" alt="js">
+                    <h3>Javascript</h3>
+                </li>
+                <li>
+                    <img src="worksimg\Ellipse 1@2x (1).png" alt="html">
+                    <h3>HTML</h3>
+                </li>
+                <li>
+                    <img src="worksimg\Ellipse 1@2x (2).png" alt="css">
+                    <h3>CSS</h3>
+                </li>
+            </ul>
+        </div>
+        <div class="frameworks">
+            <div id="intro2">
+                <h3>Frameworks</h3>
+                <button class="arrow2"><img src="images\Union@2x(1).png"></button>
+        </div>
+    </div>
+        <div id="separation"><img scr="images\Separator Bottom@2x.png"></div>
+        <div class="skills">
+            <div id="intro3">
+                <h3>Skills</h3>
+                <button class="arrow2"><img src="images\Union@2x(1).png"></button>
+        </div>
+        <div id="separation"><img scr="images\Separator Bottom@2x.png"></div>
```

Comment:
- [ ] You have done an amazing job here but I think there are some extra `spaces` on your lists, please check 👇 and see how you can improve the design to match figma design by inspecting the accurate `padding` or `margin` values :

![image](https://user-images.githubusercontent.com/99715640/212458799-80515416-013c-4348-88c3-3e2bf8b6f5da.png)